src/compare_main.cpp

```diff
@@ -89,12 +89,62 @@ void setup_compare_subcommand(CLI::App& app)
             "Use binomial model for kmer coverages [default: negative binomial]")
         ->group("Parameter Estimation");
 
+    compare_subcmd
+        ->add_flag("--dont-auto-update-params", opt->do_not_auto_update_params,
+            "By default, pandora automatically updates error rate and kmer coverage model parameters based "
+            "on the mapping of the previous sample. This could potentially generate "
+            "more accurate results if your samples have no sequencing issues and a "
+            "consistent protocol was followed for the sequencing of all samples. If this is "
+            "not the case, deactivate this feature by activating this flag")
+        ->group("Parameter Estimation");
+
     compare_subcmd
         ->add_option("--max-covg", opt->max_covg, "Maximum coverage of reads to accept")
         ->capture_default_str()
         ->type_name("INT")
         ->group("Filtering");
 
+    compare_subcmd
+        ->add_option(
+            "--min-abs-gene-coverage", opt->min_absolute_gene_coverage,
+            "Minimum absolute mean gene coverage to keep a gene. Given the "
+            "coverage on the kmers of the maximum likelihood path of a gene, we compute "
+            "the mean gene coverage and compare with the value in this "
+            "parameter. If the mean is lower than this parameter, "
+            "the gene is filtered out, e.g. if this parameter value is "
+            "3, then all genes with mean <3 will be filtered out.")
+        ->capture_default_str()
+        ->type_name("FLOAT")
+        ->group("Filtering");
+
+    compare_subcmd
+        ->add_option(
+            "--min-rel-gene-coverage", opt->min_relative_gene_coverage,
+            "Minimum relative mean gene coverage to keep a gene. This is a proportion, between 0.0 and 1.0. "
+            "Given the coverage on the kmers of the maximum likelihood path of a gene, we compute "
+            "the mean gene coverage and compare with the value in this "
+            "parameter and the global coverage. If the mean is lower"
+            " than the computed value, the gene is filtered out, e.g. if this parameter value is "
+            "0.05, then all genes with mean < 5% of the global coverage will be "
+            "filtered out.")
```

Comment:
ah i see, the internal vars directly reflect user specified vals. Which si fine, i just wonder about the defaults. it's a small thing